test/integrations/destinations/criteo_audience/network.ts

```diff
@@ -14,39 +34,74 @@ export const networkCallsData = [
           },
         },
       },
-      params: { destination: 'criteo_audience' },
-      headers: {
-        Authorization: 'Bearer success_access_token',
-        'Content-Type': 'application/json',
-        Accept: 'application/json',
-        'User-Agent': 'RudderLabs',
-      },
-      method: 'PATCH',
+      params,
+      headers,
+      method,
     },
     httpRes: { status: 200 },
   },
   {
     httpReq: {
-      url: 'https://api.criteo.com/2022-10/audiences/3485/contactlist',
+      url: 'https://api.criteo.com/2022-10/audiences/34894/contactlist',
       data: {
         data: {
           type: 'ContactlistAmendment',
           attributes: {
             operation: 'add',
-            identifierType: 'madid',
-            identifiers: ['sample_madid', 'sample_madid_1', 'sample_madid_2'],
+            identifierType: 'email',
             internalIdentifiers: false,
+            identifiers: [
+              'alex@email.com',
+              'amy@email.com',
+              'van@email.com',
+              'alex@email.com',
+              'amy@email.com',
+              'van@email.com',
+            ],
           },
         },
       },
-      params: { destination: 'criteo_audience' },
-      headers: {
-        Authorization: 'Bearer success_access_token',
-        'Content-Type': 'application/json',
-        Accept: 'application/json',
-        'User-Agent': 'RudderLabs',
+      params,
+      headers,
+      method,
+    },
+    httpRes: { status: 200 },
+  },
+  {
+    httpReq: {
+      url: 'https://api.criteo.com/2022-10/audiences/34893/contactlist',
+      data: {
+        data: {
+          type: 'ContactlistAmendment',
+          attributes: {
+            operation: 'remove',
+            identifierType: 'madid',
+            internalIdentifiers: false,
+            identifiers: [
+              'sample_madid',
```

Comment:
some data is repeated, can we do something about it?

Reply:
If any data was repeated more than twice I moved them to the common section.

Reply:
Thanks